moveit_ros/planning/trajectory_execution_manager/include/moveit/trajectory_execution_manager/trajectory_execution_manager.h

```diff
@@ -265,10 +265,14 @@ class TrajectoryExecutionManager
 
   void loadControllerParams();
 
+  double getJointAllowedStartTolerance(std::string const& jointName) const;
+  void updateJointsAllowedStartTolerance();
+
   moveit::core::RobotModelConstPtr robot_model_;
   planning_scene_monitor::CurrentStateMonitorPtr csm_;
   ros::NodeHandle node_handle_;
   ros::NodeHandle root_node_handle_;
+  ros::NodeHandle trajectory_execution_node_handle_;
```

Comment:
This extra node handle is only needed in `updateJointsAllowedStartTolerance()`.
As creating node handles is not costly, I suggest dropping this variable as a member.